### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by matviichuk92**

### CODE REVIEW

File: src/main/java/core/basesyntax/FileWork.java [hunk 1]
```diff
@@ -1,14 +1,32 @@
...
     public String[] readFromFile(String fileName) {
-        return null;
+        try {
+            List<String> lines = Files.readAllLines(Paths.get(fileName));
+            List<String> result = new ArrayList<>();
+            StringBuilder connectLines = new StringBuilder();
+            for (String value : lines) {
+                connectLines.append(value);
+            }
+            for (String value : String.valueOf(connectLines).toLowerCase().split("\\W")) {
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FileWork.java:21]:
```suggestion
            for (String value : connectLines.toString().toLowerCase().split("\\W")) {
```

File: src/main/java/core/basesyntax/FileWork.java [hunk 2]
```diff
@@ -1,14 +1,32 @@
...
+                if (value.startsWith(LETTER)) {
+                    result.add(value);
+                }
+            }
+            Collections.sort(result);
+            return result.toArray(new String[0]);
+        } catch (IOException e) {
+            throw new RuntimeException(" ");
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FileWork.java:29]: let's pass exception object to constructor too

**Comment by matviichuk92** [comment]
done

File: src/main/java/core/basesyntax/FileWork.java [hunk 3]
```diff
@@ -1,14 +1,32 @@
...
+            }
+            for (String value : connectLines.toString().toLowerCase().split("\\W")) {
+                if (value.startsWith(LETTER)) {
+                    result.add(value);
+                }
+            }
+            Collections.sort(result);
+            return result.toArray(new String[0]);
+        } catch (IOException e) {
+            throw new RuntimeException("This is mistake, sweetheart!", e);
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FileWork.java:29]:
```suggestion
            throw new RuntimeException("This is mistake, sweetheart!", e);
```
thats funny) but let's add more informative messages to exceptions

Comment (matviichuk92) [reply src/main/java/core/basesyntax/FileWork.java:29]: got it

**Pull request closed without merging**